addons/website_partner/controllers/main.py

```diff
@@ -11,13 +10,13 @@ class WebsitePartnerPage(http.Controller):
     @http.route(['/partners/<partner_id>'], type='http', auth="public", website=True)
     def partners_detail(self, partner_id, **post):
         current_slug = partner_id
-        _, partner_id = unslug(partner_id)
+        _, partner_id = self.env['ir.http']._unslug(partner_id)
```

Comment:
`self.env['ir.http']` ?

But self is a 'WebsitePartnerPage' object... no ?

Reply:
Indeed, you should use `request.env`.
This implies that there is no test/process going through this code on the runbot. We should see if it is not dead code.

Reply:
It is not dead code, but website_crm_partner_assign overrides it without call to super, to change the feature once installed (not so modular).
And probably `WebsiteMembership` too but this time by error :thinking: Same function name?

Reply:
So, I fixed that `self` => `request`.
Well seen, thank you :)